src/server/_printer.py

```diff
@@ -18,8 +18,16 @@ def print_non_standard(data):
     format = request.values.get("format", "classic")
     if format == "json":
         return jsonify(data)
+
+    if not data:
+        message = "no results"
+        result = -1
```

Comment:
[as specified in the API docs](https://cmu-delphi.github.io/delphi-epidata/api/delphi.html#response) (may need to be propagated to the compatibility mode conditional as well)

```suggestion
        result = -2
```